src/Hooks/SetupAfterCache.php

```diff
@@ -239,6 +239,7 @@ function () {
 				return new Chameleon( [
 					'name' => 'chameleon',
 					'styles' => $styles,
+					'responsive' => true,
```

Comment:
Is this a new MediaWiki feature? Since when is it available?

Reply:
From a quick git blame, it looks like emitting the viewport tag for a true `responsive` option goes back to at least 2020-08-03 (204ddf052296), and emitting the fixed-width viewport tag for the non-responsive case was first added 2 years later, in 2022-05-09 (f5afb5c48c9a).  (So, 2022-05 is when the two competing viewport tags would have started appearing in the HTML output with Chameleon.)